optimizely/decision/composite_feature_service_test.go

```diff
@@ -17,100 +17,87 @@
 package decision
 
 import (
+	"testing"
+
 	"github.com/optimizely/go-sdk/optimizely/decision/reasons"
 	"github.com/optimizely/go-sdk/optimizely/entities"
-	"github.com/stretchr/testify/assert"
-	"testing"
+	"github.com/stretchr/testify/suite"
 )
 
-func TestCompositeFeatureServiceGetDecisionFeatureExperiment(t *testing.T) {
-	mockProjectConfig := new(mockProjectConfig)
-	testFeatureDecisionContext := FeatureDecisionContext{
+type CompositeFeatureServiceTestSuite struct {
+	suite.Suite
+	mockFeatureService         *MockFeatureDecisionService
+	mockFeatureService2        *MockFeatureDecisionService
+	testFeatureDecisionContext FeatureDecisionContext
+}
+
+func (s *CompositeFeatureServiceTestSuite) SetupTest() {
+	mockConfig := new(mockProjectConfig)
+
+	s.mockFeatureService = new(MockFeatureDecisionService)
+	s.mockFeatureService2 = new(MockFeatureDecisionService)
+
+	// Setup test data
+	s.testFeatureDecisionContext = FeatureDecisionContext{
 		Feature:       &testFeat3335,
-		ProjectConfig: mockProjectConfig,
-	}
-	testExperimentDecisionContext := ExperimentDecisionContext{
-		Experiment:    &testExp1113,
-		ProjectConfig: mockProjectConfig,
-	}
-	mockExperimentDecision := ExperimentDecision{
-		Decision:  Decision{reasons.BucketedIntoVariation},
-		Variation: &testExp1113Var2223,
+		ProjectConfig: mockConfig,
 	}
+}
 
+func (s *CompositeFeatureServiceTestSuite) TestGetDecision() {
```

Comment:
Do we need additional test coverage for `NewCompositeFeatureService`, `NewFeatureExperimentService`, or `NewRolloutService`?

Reply:
Yeah I can add test coverage for it